### DISCUSSION TITLE
HA Split Configuration (Missing parts/pieces)

### DISCUSSION THREAD

**Discussion by fireheadman** [frenck/home-assistant-config#78] [Q&A] [answered]
Frenck...   Just watched a youtube video you posted a while back on splitting HA configs.
I was attempting to model your configuration since you posted it here (github).

Please let me know if there are hidden objects/folders.. etc in your repo?
I'm having a difficult time following everything

For example  (this references the ../entities/sensors  directory)
![image](https://user-images.githubusercontent.com/19847195/130502272-22ab06e1-7e78-43e6-8433-8e2078f8e5c6.png)

....but when I navigate your tree, I do not see the sensors directory.
So how is this working for you?
![image](https://user-images.githubusercontent.com/19847195/130502438-e0f318a1-ea12-40e8-9a94-35c62eade2a0.png)

Would you be able to upload your current configuration so the missing parts/pieces are present?
THanks
Ben

**Comment by niro1987** [comment]
Have a look at mine, I've also used @frenck 's model.

The sensors folder might just be empty a.t.m.

**Comment by fireheadman** [comment]
thanks!... much more content, makes it easier to see the big picture.

**Comment by frenck** [comment, marked as answer]
My sensor folder is empty. I currently don't have any manual sensors defined (I don't have a need for it right now).
